.github/workflows/lint.yml

```diff
@@ -46,5 +46,5 @@ jobs:
 
       - name: Lint
         run: |
-          nvim -l scripts/format-queries.lua
+          nvim -l scripts/format-queries.lua queries
```

Comment:
```suggestion
          nvim -l scripts/format-queries.lua
```

You need to rebase; I changed the script to default to current dir.

Reply:
It also tries to format the test file, so I deliberately do that again

Reply:
Maybe there's a better solution here :thinking:

Reply:
Decided to `; format-ignore` pilled that file